<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>데이터 수집 Agent</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_home_">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_home_ {
    border: 0px;
	background-color: rgb(107, 106, 108);
}

QPushButton#btn_home_:hover {
    background-color: rgb(130, 130, 130)
}

QPushButton#btn_home_:pressed {
    background-color: rgb(169, 167, 171);
}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_path_">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_path_ {
    border: 0px;
	background-color: rgb(107, 106, 108);
}

QPushButton#btn_path_:hover {
   background-color: rgb(130, 130, 130)
}

QPushButton#btn_path_:pressed {
    background-color: rgb(169, 167, 171);
}</string>
    </property>
    <property name="text">
     <string>환경설정</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_info_image_">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>72</y>
      <width>460</width>
      <height>137</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>137</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>137</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="grp_path_">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>540</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 3px;
}</string>
    </property>
    <property name="title">
     <string>저장 경로</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="lbl_path_">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>35</y>
       <width>105</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>로그 저장 디렉토리 :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ed_path_">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>35</y>
       <width>335</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gray;
border-radius: 10px;
color: black;
background-color: rgb(243, 240, 246)
</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_dot_">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>35</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton#btn_dot_ {
    border: 2px solid gray;
    border-radius: 10px;
}

QToolButton#btn_dot_:hover {
    border: 2px solid gray;
    border-radius: 10px;
    background-color: rgb(107, 106, 108);
}

QToolButton#btn_dot_:pressed {
    border: 2px solid gray;
    border-radius: 10px;
    background-color: rgb(169, 167, 171);
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_cancle_">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_cancle_ {
    border: 2px solid gray;
    border-radius: 10px;
}

QPushButton#btn_cancle_:hover {
    border: 2px solid gray;
    border-radius: 10px;
    background-color: rgb(107, 106, 108);
}

QPushButton#btn_cancle_:pressed {
    border: 2px solid gray;
    border-radius: 10px;
    background-color: rgb(169, 167, 171);
}</string>
    </property>
    <property name="text">
     <string>취소</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ok_">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>180</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_ok_ {
    border: 2px solid gray;
    border-radius: 10px;
}

QPushButton#btn_ok_:hover {
    border: 2px solid gray;
    border-radius: 10px;
    background-color: rgb(107, 106, 108);
}

QPushButton#btn_ok_:pressed {
    border: 2px solid gray;
    border-radius: 10px;
    background-color: rgb(169, 167, 171);
}</string>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_info_">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_info_ {
    border: 0px;
	background-color: rgb(107, 106, 108);
}

QPushButton#btn_info_:hover {
    background-color: rgb(130, 130, 130)
}

QPushButton#btn_info_:pressed {
    background-color: rgb(169, 167, 171);
}</string>
    </property>
    <property name="text">
     <string>Information</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>0</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(107, 106, 108);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>0</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(107, 106, 108);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_on_off_">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>160</y>
      <width>69</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="farm_resource.qrc">
      <normaloff>:/imgs/imgs/btn_off.png</normaloff>
      <normalon>:/imgs/imgs/btn_on.png</normalon>
      <disabledoff>:/imgs/imgs/btn_off.png</disabledoff>
      <disabledon>:/imgs/imgs/btn_on.png</disabledon>
      <activeoff>:/imgs/imgs/btn_off.png</activeoff>
      <activeon>:/imgs/imgs/btn_on.png</activeon>
      <selectedoff>:/imgs/imgs/btn_off.png</selectedoff>
      <selectedon>:/imgs/imgs/btn_on.png</selectedon>:/imgs/imgs/btn_off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>69</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_start_">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>135</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>시작 프로그램 등록</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_auto_start_">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>155</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>프로그램 시작 시 자동 실행</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_run_">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>56</y>
      <width>88</width>
      <height>88</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>88</width>
      <height>88</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>88</width>
      <height>88</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_stop_">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>56</y>
      <width>88</width>
      <height>88</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>88</width>
      <height>88</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>88</width>
      <height>88</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_text_">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>160</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>데이터 수집 Agent가 실행 중 입니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_current_">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="farm_resource.qrc"/>
 </resources>
 <connections/>
</ui>
